bundles/org.openhab.binding.revogi/src/main/java/org/openhab/binding/revogi/internal/RevogiSmartStripControlHandler.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.revogi.internal;
+
+import static org.openhab.core.library.unit.MetricPrefix.MILLI;
+import static org.openhab.core.library.unit.SmartHomeUnits.AMPERE;
+import static org.openhab.core.library.unit.SmartHomeUnits.WATT;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.revogi.internal.api.Status;
+import org.openhab.binding.revogi.internal.api.StatusService;
+import org.openhab.binding.revogi.internal.api.SwitchService;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link RevogiSmartStripControlHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Andi Bräu - Initial contribution
+ */
+@NonNullByDefault
+public class RevogiSmartStripControlHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(RevogiSmartStripControlHandler.class);
+    private final StatusService statusService;
+    private final SwitchService switchService;
+    private @Nullable ScheduledFuture<?> pollingJob;
+
+    private @Nullable RevogiSmartStripControlConfiguration config;
+
+    public RevogiSmartStripControlHandler(Thing thing, StatusService statusService, SwitchService switchService) {
+        super(thing);
+        this.statusService = statusService;
+        this.switchService = switchService;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        switch (channelUID.getId()) {
```

Comment:
Can you handle the REFRESH command?

Reply:
now it is handled, does it mean anything else?

Reply:
Looks good.